mozilla-release/browser/locales/en-US/chrome/browser/ghosteryPrivateBrowsing.dtd

```diff
@@ -0,0 +1,14 @@
+<!--
+# Copyright Cliqz GmbH, 2016.
```

Comment:
:)

Reply:
@luciancor I left it there intentionally